src/decrement_lookup_table.h

```diff
@@ -0,0 +1,1114 @@
+/**
+ * @file    LookUpTable.h
+ * @author  Thomas Lewiner <thomas.lewiner@polytechnique.org>
+ * @author  Math Dept, PUC-Rio
```

Comment:
@andrewykhong does this file have a license?

Reply:
I'm not sure. This was implemented before this PR. I believe Arnaud added this to SPARTA? @aborner1

Reply:
Well we've had the original lookup_table.h file in there for years. I had emailed Thomas Lewiner in 2019, and here is a snippet of his response:
```
Regarding the Marching Cubes license, it is 100% free (even no GPL: I did not know which one to choose at the time...).
```

Reply:
@aborner1 thanks, I added this to the file.